How do I use phf_codegen with the key being a byte string? (create example)

I'm not finding any docs or examples that cover various kinds of keys, particularly byte strings.

When I take my input string and do `as_bytes` on it into `entry` I get the following errors
```
error[E0308]: mismatched types
    --> /home/epage/git/defenestrate/target/debug/build/defenestrate-73941c0b26a3ba8c/out/codegen.rs:6834:10
     |
6834 |         ([109, 105, 115, 100, 101, 109, 101, 97, 110, 101, 114], "misdemeanor"),
     |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |          |
     |          expected &[u8], found array of 11 elements
     |          help: consider borrowing here: `&[109, 105, 115, 100, 101, 109, 101, 97, 110, 101, 114]`
```

--- a/phf_shared/src/lib.rs
+++ b/phf_shared/src/lib.rs
@@ -88,7 +88,6 @@ macro_rules! delegate_debug(
 );
 
 delegate_debug!(str);
-delegate_debug!([u8]);
 delegate_debug!(char);
 delegate_debug!(u8);
 delegate_debug!(i8);
@@ -141,6 +140,14 @@ impl PhfHash for [u8] {
     #[inline]
     fn phf_hash<H: Hasher>(&self, state: &mut H) {
         state.write(self);
+    }
+}
+
+impl FmtConst for [u8] {
+    #[inline]
+    fn fmt_const(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // slices need a leading reference
+        write!(f, "&{:?}", self)
     }
 }
 

